### PR TITLE
Add loading dialog after sending Circles

### LINKED ISSUE #590: Transferring circles message
-Transfer can take long sometimes, therefore we will implement a modal with a msg asking for patient and displaying a loading animation. 

-here design material: https://www.figma.com/file/ZTqF54s6CSHTwAxp0U0JjQ/Circles-myxogastria?node-id=8692%3A22818&t=NN5XyKajk1K4swZU-1

### PR DESCRIPTION
**Pull request by mikozet**

### CODE REVIEW

**Comment by louilinn** [comment]
Sorry to say you broke the confirm transfer dialogue. Fix that and I'll check the code :)

## Dialogue state here
Screenshot from 2023-02-10 16-46-52

## Dialogue state on main
Screenshot from 2023-02-10 16-43-32

## Waiting - looks fine
Screenshot from 2023-02-10 16-43-39

File: src/views/SendConfirm.js [hunk 1]
```diff
@@ -197,54 +181,15 @@ const SendConfirm = () => {
...
-            </Button>
-          </Box>
-          <Box maxWidth="60%" mb={2} mx="auto">
-            <Button fullWidth isOutline onClick={handleConfirmClose}>
-              {translate('SendConfirm.dialogSendCancel')}
-            </Button>
-          </Box>
-        </DialogContent>
-      </Dialog>
+      <SendConfirmDialog
```
Comment (louilinn) [inline src/views/SendConfirm.js:184]: needs username

File: src/components/SendConfirmDialog.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+};
+
+SendConfirmDialog.propTypes = {
+  amount: PropTypes.string,
+  handleConfirmClose: PropTypes.func,
+  handleSend: PropTypes.func,
+  isConfirmationShown: PropTypes.bool,
+  isLoadingConfirmationShown: PropTypes.bool,
+  paymentNote: PropTypes.string,
+  receiver: PropTypes.string,
```
Comment (louilinn) [inline src/components/SendConfirmDialog.js]: required

File: src/views/SendConfirm.js [hunk 3]
```diff
@@ -197,54 +181,15 @@ const SendConfirm = () => {
...
+        amount={amount}
+        handleConfirmClose={handleConfirmClose}
+        handleSend={handleSend}
+        isConfirmationShown={isConfirmationShown}
+        isLoadingConfirmationShown={isLoadingConfirmationShown}
+        paymentNote={paymentNote}
+        sender={sender}
```
Comment (louilinn) [inline src/views/SendConfirm.js:200]: and receiver

**Review by louilinn — CHANGES_REQUESTED** [review]
See comments

**Comment by louilinn** [comment]
I fixed the issues mentioned in my review
- [x] receiver username displaying
- [x] cancel button behavior fixed
- [x] clicking outside the dialog while loading does not close dialog
- [x] replaced too techy read more url with generic FAQ url
- [x] rebased (fixed button styling)

What remains
- [ ] get a better url and add it using constants.js

**Comment by louilinn** [comment]
@mikozet pls check my changes :)

**Comment by mikozet** [comment]
@louilinn loos good to me :) - I am an author so I cannot approve

**Review by louilinn — APPROVED** [review]

**Pull request merged**